tests/karma/js-unit.conf.js

```diff
@@ -58,6 +58,7 @@ module.exports = function (config) {
 			'resources/wikia/modules/thumbnailer.js',
 			'resources/wikia/modules/uniqueId.js',
 			'resources/wikia/modules/scrollToLink.js',
+			'resources/wikia/modules/iframeWriter.js',
```

Comment:
Let's sort them as well, seems like only scrollToLink and iframeWriter are out of order.